doc/classes/ProjectSettings.xml

```diff
@@ -193,6 +193,12 @@
 		</method>
 	</methods>
 	<members>
+		<member name="app_protocol/enable_app_protocol" type="bool" setter="" getter="" default="false">
+			Is this system enabled at all
```

Comment:
For consistency with the other properties, I guess it should be like `If [code]true[/code], app protocol handling is enabled.`

Reply:
I think to me we need to make it more presentable to a user, since they won't swap this option on using code.
`Is AppProtocol enabled for the application, e.g. being able to launch yourappname:// from URL`